src/test/functional/features/joint-final-order.feature

```diff
@@ -0,0 +1,32 @@
+Feature: Joint final order
+
+  Background: Logged in for hub page
+    Given I create a new user and login
+    And I've already completed the form using the fixture "jointApplicant1CompleteCase"
+    When I go to "/"
+    And I click "Send for review"
+    Then the page URL should be "/application-sent-for-review"
+    When I enter my valid case reference and valid access code
+    Then the page should include "You need to review your joint application"
+    Given I've already completed the form using the fixture "jointApplicant2CompleteCase" for "applicant2"
+    And I go to "/applicant2/confirm-your-joint-application"
+    Given I select "I confirm that I’m applying to the court to dissolve my marriage (get a divorce)"
+    And I select "I believe that the facts stated in this application are true"
+    When I click "Submit"
+    Then the page URL should be "/applicant2/needs-to-confirm-joint-application"
+
+#  //TODO when final order is implemented change test to user path instead of changing the states for each page
```

Comment:
Might be best to make a ticket for this and add the number here so we don't forget about this.

Reply:
Refactored the tests into the hub page feature tests, so this TODO should not be necessary.